aten/src/ATen/native/tags.yaml

```diff
@@ -40,3 +40,8 @@
           type promotion and boardcasting ops.
           Canonical aten ops is also effectively the opset produced by torchdynamo.export(aten_graph=True),
           and thus can be used as an opset for export purpose.
+- tag: pointwise
```

Comment:
CC @wanchaol, is there way to remove https://github.com/pytorch/pytorch/blob/master/torch/distributed/_tensor/ops/pointwise_ops.py#L36 and use the tag ? 

I see elsewhere there is `Below ops are some pointwise ops that are yet to be supported` so I guess you need to differentiate between supported pointwise op and unsupported pointwise ops

Reply:
This is nice! Thanks for adding the tag! I have been thinking about tagging a while ago and wonder if DTensor could just simply utilize the tags. I think it should be doable, I'll try it out once you land the diff, thanks!